homeassistant/components/microbees/entity.py

```diff
@@ -0,0 +1,51 @@
+"""Base entity for microBees."""
+
+from microBeesPy.microbees import Actuator, Bee, MicroBees
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+
+
+class MicroBeesEntity(CoordinatorEntity[MicroBeesUpdateCoordinator]):
+    """Base class for microBees entities."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: MicroBeesUpdateCoordinator,
+        act: Actuator,
+        bee: Bee,
```

Comment:
Again, why do we put in the full actuator and bee instead of just the id?

Reply:
Because we need to set the name and device info properties (model, name)

Reply:
You can just access the property in the constructor